KlaviyoV3Sdk/KlaviyoV3Api.php

```diff
@@ -396,8 +394,7 @@ public function buildEventProperties($eventProperties, $time, $metric)
                 self::DATA_KEY_PAYLOAD => array(
                     self::TYPE_KEY_PAYLOAD => self::METRIC_KEY_PAYLOAD,
                     self::ATTRIBUTE_KEY_PAYLOAD => array(
-                        self::NAME_KEY_PAYLOAD => $metric,
-                        self::SERVICE_PAYLOAD_KEY => self::MAGENTO_TWO_INTEGRATION_SERVICE_KEY
```

Comment:
removed this, its wrong!

Reply:
a) we don't want this branded, which is what service does
b) the integration key is incorrect anyways - magento_two not magentotwo